src/pyhf/optimize/opt_minuit.py

```diff
@@ -102,8 +105,11 @@ def _minimize(
         # 0: Fast, user-provided gradient
         # 1: Default, no user-provided gradient
         strategy = options.pop(
-            'strategy', self.strategy if self.strategy is not None else not do_grad
+            'strategy', self.strategy if self.strategy is not None else int(not do_grad)
```

Comment:
As the `iminuit.Minuit.strategy` is an integer, instead of having to have a human determine the integer representation of the inverse of the truthiness of `do_grad` during debugging or inspection do this in code.